DATA/production/configurations/asyncReco/setenv_extra.sh

```diff
@@ -671,6 +678,10 @@ if [[ $PERIOD == "LHC22c" ]] || [[ $PERIOD == "LHC22d" ]] || [[ $PERIOD == "LHC2
   export ARGS_EXTRA_PROCESS_o2_aod_producer_workflow+=" --ctpreadout-create 1"
 fi
 
+if [[ $ALIEN_JDL_LPMANCHORYEAR == "2023" ]] && [[ $BEAMTYPE == "PbPb" ]] ; then
```

Comment:
Hello @shahor02 ,
Please remove this. Or even better change it to (so I don't open a PR myself):

```
if [[ $ALIEN_JDL_THISAODS == "1" ]] ; then
...
```
Thanks!

Reply:
OK, done.